# HG changeset patch
# User HackaLittleBit
# Date 1460905705 -3600
#      Sun Apr 17 16:08:25 2016 +0100
# Branch trunk
# Node ID 2a6c59a1306d39ef24ddc3547b1a4d560dcc4d1b
# Parent  6dee60f442384f92de07fa8c6ae025d7a9644723
Add: Don't leave planes stuck on platform

diff -r 6dee60f44238 -r 2a6c59a1306d src/aircraft_cmd.cpp
--- a/src/aircraft_cmd.cpp	Sat Apr 16 16:59:34 2016 +0100
+++ b/src/aircraft_cmd.cpp	Sun Apr 17 16:08:25 2016 +0100
@@ -1973,6 +1973,7 @@
 				AddVehicleAdviceNewsItem(STR_NEWS_AIRCRAFT_DEST_TOO_FAR, v->index);
 			}
 		}
+		v->current_order.Free();
 		return;
 	}
 
